src/index.html

```diff
@@ -6,9 +6,351 @@
+            <div class="hobbies">
+              <i class="fas fa-utensils"></i>
+              <p>BBQ</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-glass-cheers"></i>
+              <p>Party</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-ad"></i>
+              <p>Cinema</p>
+            </div>
+            <div class="hobbies">
+              <i class="fas fa-cookie"></i>
+              <p>Cooking</p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="dot-sections">
+        <span class="dot" onclick="currentSlide(1)"></span>
+        <span class="dot" onclick="currentSlide(2)"></span>
+        <span class="dot" onclick="currentSlide(3)"></span>
+        <span class="dot" onclick="currentSlide(4)"></span>
+      </div>
+    </div>
+
+    <!-- Skill Section-->
+    <div class="skills-section">
+      <h2>SKILLS</h2>
+      <div class="skill">
+        <p class="skill_name">Microsoft Excel</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Microsoft Word</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Adobe</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">PowerPoint</p>
+        <p class="skill_level">Proficient</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">French</p>
+        <p class="skill_level">Native</p>
+      </div>
+      <div class="skill">
+        <p class="skill_name">Spanish</p>
+        <p class="skill_level">Advanced</p>
+      </div>
+    </div>
+
+    <!-- Contacts Section-->
+    <div class="contacts-section">
+      <div class="id-section">
+        <div class="title">
+          <h2>QUICK ID</h2>
+        </div>
+        <div class="border"></div>
+        <div class="contact_details">
+          <table>
+            <tr>
+              <td>Phone</td>
+              <td>123-456-7890</td>
+            </tr>
+            <tr>
+              <td>Email</td>
+              <td><a href="mailto:info@mysite.com">info@mysite.com</a></td>
+            </tr>
+            <tr>
+              <td>Website</td>
+              <td><a href="" target="_blank">www.wix.com</a></td>
+            </tr>
+            <tr>
+              <td>Address</td>
+              <td>500 Terry Francois Street</td>
+            </tr>
+            <tr>
+              <td></td>
+              <td>San Francisco, CA 94158</td>
+            </tr>
+          </table>
+        </div>
+      </div>
+
+      <div class="contact-me-section">
+        <div class="title">
+          <h2>CONTACT ME</h2>
+        </div>
+        <div class="border"></div>
+        <div class="contact_me_inputs">
+          <form action="" targer="_blank" method="post">
+            <input type="text" placeholder="Name" />
```

Comment:
No `label`, just placeholder, this is bad usability and accessibility.

no `name` attribute, this form is not functional.